source/_integrations/shelly.markdown

```diff
@@ -49,6 +49,19 @@ Integrate [Shelly devices](https://shelly.com) into Home Assistant.
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+Host:
+    description: "The Hostname or IP address of your Shelly device. You can find it in your router."
+Port:
+    description: "Custom TCP port of the device. Change this only if the device is connected via Shelly Range Extender."
+{% endconfiguration_basic %}
+
+## Removing the integration
```

Comment:
In another PR I was asked to put the removing instruction at the end of the doc.